### ISSUE TITLE
User should be able to deselect/select a single lot

### ISSUE THREAD

**Issue by andycochran** [NYCPlanning/labs-applicantmaps#275]
In lot mode, you can’t deselect a lot when it’s the only lot selected. If you click the selected lot, it stays highlighted. However, I assume that (even though it’s highlighted) it’s not actually part of the selection, because when you click another lot the previously selected lot is deselected. So, if you click a single selected lot multiple times, you don’t know if it’s selected or not until you click some other lot. Then, whether or not it stays selected depends on if you’ve clicked it an even or odd number of times. 

This bug will be frustrating in this use case:
- User has a project that’s one lot
- They select the wrong one
- They try to deselect it before selecting the right one
- They click it a number of times, becoming frustrated that the wrong lot is still selected

**Comment by andycochran** [comment]
@allthesignals added a skipped test for this.

**Comment by hannahkates** [comment]
Duplicate of #419
